notebooks/4_1_software_framework.ipynb

```diff
@@ -232,61 +213,11 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": null,
    "metadata": {
     "tags": []
    },
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "\u001b[1;31mDocstring:\u001b[0m\n",
-       "Specify a compute tile C++ kernel and return an npu.build.Kernel object.\n",
-       "\n",
-       "This cell magic command allows users to input C++ kernel code within\n",
-       "a Jupyter notebook cell. It then returns a corresponding Kernel object\n",
-       "that can be compiled into an object file for use in a Riallto application.\n",
-       "The Cpp source must return a void type, input and output buffers are specified\n",
-       "as pointer types, as parameters are specified with non-pointer types.\n",
-       "\n",
-       "Header files included in the directory where the notebook is are permitted.\n",
-       "\n",
-       "Parameters\n",
-       "----------\n",
-       "cell : str\n",
-       "    The string content of the cell, expected to be C++ code defining\n",
-       "    the kernel.\n",
-       "\n",
-       "Returns\n",
-       "-------\n",
-       "Kernel : object\n",
-       "    Returns a Kernel object that has the same name as the last function\n",
-       "    defined in the cell magic. See npu.build.Kernel.\n",
-       "\n",
-       "Examples\n",
-       "--------\n",
-       "In a Jupyter notebook %%kernel cell\n",
-       "\n",
-       "\n",
-       "void passthrough(uint8_t *in_buffer, uint8_t* out_buffer, uint32_t nbytes) {\n",
-       "    for(int i=0; i<nbytes; i++) {\n",
-       "        out_buffer[i] = in_buffer[i];\n",
-       "        \n",
-       "    }\n",
-       "    \n",
-       "}\n",
-       "\n",
-       "\n",
-       "\n",
-       "This will construct a passthrough npu.build.Kernel object that can be used within\n",
-       "a callgraph to construct a complete application. \n",
-       "\u001b[1;31mFile:\u001b[0m      c:\\users\\riallto\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\npu\\magic.py"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
-    }
-   ],
+   "outputs": [],
```

Comment:
Is it possible to bring back this text?

Reply:
Ah sorry, yes my bad.

Reply:
Added this stuff again if you want to take a look.

Reply:
I accidentally comited the wrong notebook before, should be there now.